pkg/cli/defaults/defaults.go

```diff
@@ -25,7 +25,7 @@ func Set(clx *cli.Context, dataDir string) error {
 	cmds.ServerConfig.HTTPSPort = 6443
 	cmds.ServerConfig.APIServerPort = 6443
 	cmds.ServerConfig.APIServerBindAddress = "0.0.0.0"
-	cmds.ServerConfig.DisableKubeProxy = true
+	cmds.ServerConfig.DisableKubeProxy = false
```

Comment:
just delete it
```suggestion
```

Reply:
I am planning to wait until kube proxy is added to the kubernetes-images. If we wait on that, then I don't need to make the changes in the first two items, correct?

Reply:
Yeah, if you get it in the kubernetes image then this should be fine